docs/source/v1.2.md.inc

```diff
@@ -2,6 +2,8 @@
 
 ### :new: New features & enhancements
 
+- mne.get_datatype is now cached to reduce computation time when large datasets are being processed. (#720 by @apmellot)
```

Comment:
Very nice! I always like to have things a little bit less technical so users will get a feel for whether a change is going to help them or not. How about this:

```suggestion
- We improved caching to reduce initialization time when large datasets are being processed. (#720 by @apmellot)
```